sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -0,0 +1,543 @@
+
+Install the Azure Text Analytics client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/ai-text-analytics
+```
+
+### Create and authenticate a `TextAnalysisClient`
+
+To create a client object to access the Language API, you will need the `endpoint` of your Language resource and a `credential`. The Text Analytics client can use either Azure Active Directory credentials or an API key credential to authenticate.
+
+You can find the endpoint for your Language resource either in the [Azure Portal][azure_portal] or by using the [Azure CLI][azure_cli] snippet below:
+
+```bash
+az cognitiveservices account show --name <your-resource-name> --resource-group <your-resource-group-name> --query "properties.endpoint"
+```
+
+#### Using an API Key
+
+Use the [Azure Portal][azure_portal] to browse to your Language resource and retrieve an API key, or use the [Azure CLI][azure_cli] snippet below:
+
+**Note:** Sometimes the API key is referred to as a "subscription key" or "subscription API key."
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+Once you have an API key and endpoint, you can use the `AzureKeyCredential` class to authenticate the client as follows:
+
+```javascript
+const { TextAnalysisClient, AzureKeyCredential } = require("@azure/ai-text-analytics");
+
+const client = new TextAnalysisClient("<endpoint>", new AzureKeyCredential("<API key>"));
+```
+
+#### Using an Azure Active Directory Credential
+
+Client API key authentication is used in most of the examples, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below,
+or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and grant access to Language by assigning the `"Cognitive Services User"` role to your service principal (note: other roles such as `"Owner"` will not grant the necessary permissions, only `"Cognitive Services User"` will suffice to run the examples and the sample code).
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```javascript
+const { TextAnalysisClient } = require("@azure/ai-text-analytics");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const client = new TextAnalysisClient("<endpoint>", new DefaultAzureCredential());
+```
+
+## Key concepts
+
+### TextAnalysisClient
+
+`TextAnalysisClient` is the primary interface for developers using the Text Analytics client library. Explore the methods on this client object to understand the different features of the Language service that you can access.
```

Comment:
yeah I think the naming should be coherent throughout the doc, maybe we stop using the word `analytics` ?

Reply:
would it be confusing that the name of the package is `ai-text-analytics` but the client name is different?

Reply:
@kristapratico I liked the new name better and want to pitch it to the architects. The reason simply being  `analysis`  feel closer to `analyze` so `TextAnalysisClient.analyze` is more natural imo than `TextAnalyticsClient.analyze`. That being said, I understand the confusion concern since `analytics` is still part of the package name.

Do you think it could help if we replace `analytics` with `analysis` in the package description?

Reply:
Is there a problem with keeping the old name for the client or is this more of a rebranding decision?

Reply:
No problem per se, this is mostly me trying hard to break with the old and rethinking everything.

Reply:
Wasn't our agreed upon goal to support the Language service while keeping the existing API shape?   It seems like JS is moving more in the direction of drifting from that goal.   Are we not concerned about losing alignment between the languages?

Reply:
I understand the alignment concern and I talked to the service team about whether they feel comfortable having breaking changes in JS and they're ok with it because most of their customers are coming from the other languages.

For context, most of the existing API shape is still there. I did minimal renames mainly to correct things that should have been named differently to conform to specific patterns that all languages are using. The only other rename is for the client class and I think that proposition has merit. In addition to my subjective opinion above, the old name was following the service's old name and there is no need to continue using it.

Reply:
I assume this is intended to be specific to JS?

Reply:
I am proposing this rename for JS only for this time, but I would like your input on it nonetheless so we all could align on it if/when you decide to redesign your client in the future.

Reply:
I've no issues with the name itself; I would have chosen the same for a new iteration of the client.   My concern is more "lets not rename the client until we reach that point in the arbitrary future where we want to shift to new API patterns" - which is not in the near-term for .NET nor, I believe, languages other than JS.

Reply:
Thanks folks for sharing your thoughts! May I suggest that we take this up in the archboard if there is still appetite to pursue alignment further? I am hopeful that JS can use the new name now and other languages to follow suit once they're ready.